### ISSUE TITLE
cloudlaunch: use Create Project API

### ISSUE THREAD

**Issue by bradfitz** [camlistore/camlistore#819]
It seems there's a programmatic way to create Google Cloud Platform projects now?

See:
https://github.com/google/google-api-go-client/issues/138

(the Cloud Resource Manager API, as of v1beta1?)

**Comment by bradfitz** [comment]
This API is now officially launched and out of beta.

Please prioritize using this in our launcher.

**Comment by mpl** [comment]
ACK, top of TODO list now.

**Comment by mpl** [comment]
@bradfitz while working on this, I found out that I was getting a 403 when trying to create a new project, using a service account from one of my existing projects to authenticate (the same way we would use the camlistore-website service account in production). I believe the reason is, according to https://cloud.google.com/resource-manager/ , that project creation is invite-only, even using the API.

So I've contacted the sales team, as they request but, even if I do get an invite, and we do get one for the camlistore-website service account, unless I got confused by OAuth2 again, don't we need each user to have that privilege as well, since they are the ones delegating to us whatever the launcher is doing?
